src/preact.js

```diff
@@ -5,10 +5,13 @@ import { render } from './render';
 import { rerender } from './render-queue';
 import options from './options';
 
+const createRef = Object;
```

Comment:
Just curious: Is using `Object` faster than a literal `{}`?

Reply:
slower. I just benchmarked and while I'd originally thought it'd be worth using Object here, the performance hit is surprising (50%):
https://esbench.com/bench/5b1d4021f2949800a0f61d25
<img width="372" src="https://user-images.githubusercontent.com/105127/41203064-3e48adac-6ca0-11e8-8d12-cbef5edf5ab3.png">